src/lambdas/handlers/breeds-get/breeds-get.test.ts

```diff
@@ -40,20 +40,34 @@ describe('breeds-get handler', () => {
     })
   })
 
-  it('returns a status of 500 when a timeout error occurs', async () => {
+  it('returns a status of 500 when an error occurs', async () => {
     mockedFetch.mockReturnValueOnce({
       json: () => {
-        setTimeout(() => {
-          throw new Error('timeout occurred')
-        }, 1000 * 10)
+        throw new Error('oops')
       },
     })
 
     const response = await handler()
 
     expect(response).toMatchObject({
       statusCode: 500,
-      message: `Unsuccessful response from '/breeds/list/all'`,
+      message: `Request Failed, Error: {}`,
+    })
+  })
+
+  it.only('returns a status of 500 when a timeout error occurs', async () => {
```

Comment:
```suggestion
  it('returns a status of 500 when a timeout error occurs', async () => {
```
Left over artifact from testing? Good to double check that you remove any skips/only between pushing updates.

Reply:
Just a miss :(

Reply:
No worries, I'd be lying if I said I never did it lol